consistency-checker/src/test/java/com/redhat/lightblue/migrator/consistency/MigrationJobTest.java

```diff
@@ -139,6 +216,109 @@ public void testDocumentsConsistent_SourceAndDestinationNotConsistent(){
         assertFalse(migrationJob.documentsConsistent(sourceNode, destNode));
     }
 
+    /**
+     * Technically the two values do not match, however the field in question
+     * is excluded and should pass.
+     */
+    @Test
+    public void testDocumentsConsistent_WithMultiLevelExclude(){
+        JsonNodeFactory factory = JsonNodeFactory.withExactBigDecimals(false);
+
+        ObjectNode sourceChild = factory.objectNode();
+        sourceChild.put("someChildKey", "hi");
+
+        ObjectNode sourceNode = factory.objectNode();
+        sourceNode.put("somekey", sourceChild);
+
+        ObjectNode destChild = factory.objectNode();
+        destChild.put("someChildKey", "there");
+
+        ObjectNode destNode = factory.objectNode();
+        destNode.put("somekey", destChild);
+
+        MigrationConfiguration jobConfiguration = new MigrationConfiguration();
+        jobConfiguration.setComparisonExclusionPaths(Arrays.asList("somekey.someChildKey"));
+        migrationJob.setJobConfiguration(jobConfiguration);
+
+        assertTrue(migrationJob.documentsConsistent(sourceNode, destNode));
+    }
+
+    /**
+     * Ensures that a more complex json document, with a deep invalid field but where that field is excluded.
+     * Should pass.
+     */
+    @Test
+    public void testDocumentsConsistent_ComplexJson_Pass() throws Exception{
```

Comment:
A few things to make round out the tests, add some that:
* have a different value for "something" at the top level, verify docs are not consistent
* exclude "something" instead of "array.something" and verify documents are not consistent